site/app/controllers/admin/UsersController.php

```diff
@@ -892,7 +901,11 @@ public function uploadUserList($list_type = "classlist") {
             'grader_group' => 'Grader-level should be in between 1 - 4.',
             'user_password' => 'user_password cannot be blank',
             'user_preferred_firstname' => 'Preferred first name must be alpha characters, white-space, or certain punctuation.',
-            'user_preferred_lastname' => 'Preferred last name must be alpha characters, white-space, or certain punctuation.'
+            'user_preferred_lastname' => 'Preferred last name must be alpha characters, white-space, or certain punctuation.',
+            'grading_assignments_format' => 'Grading assignments must be comma-separated course registration sections, ' .
+                'enclosed in double quotes (e.g. "1,3,STAFF").',
```

Comment:
Tested and it looks like it needs to be enclosed in three quotation marks for it to work

Reply:
Fixed it so that it's only enclosed in one! Thanks for the catch :)

Sorry I parsed it wrong due to my inaccurate test files. I had manually added double quotes to the column in Excel, but Excel already automatically adds double quotes to comma-inclusive columns when exporting to CSV.